.github/workflows/build-publish.yaml

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+on:
+  push:
+    # branches:
+    #   - master
+    paths-ignore:
+      - '**.md'
+jobs:
+  build:
+    env:
+      IMAGE_NAME: quay.io/eclipse/che-backup-server-rest
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Docker login
+        uses: azure/docker-login@v1
+        with:
+          login-server: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      - name: Docker Buildx
+        uses: crazy-max/ghaction-docker-buildx@v1.5.0
+        with:
+          version: v0.4.1
```

Comment:
please use the new location: https://github.com/crazy-max/ghaction-docker-buildx#moved-to-docker-organization

Reply:
Thanks, will do